src/Simplify/AstHelpers.elm

```diff
@@ -71,12 +71,11 @@ removeParensFromPattern patternNode =
 
 isSpecificValueOrFunction : ( ModuleName, String ) -> ModuleNameLookupTable -> Node Expression -> Bool
 isSpecificValueOrFunction ( moduleName, name ) lookupTable expressionNode =
-    case removeParens expressionNode of
-        Node fnRange (Expression.FunctionOrValue _ foundName) ->
-            (foundName == name)
-                && (ModuleNameLookupTable.moduleNameAt lookupTable fnRange == Just moduleName)
+    case getSpecificValueOrFunction ( moduleName, name ) lookupTable expressionNode of
```

Comment:
For a small perf improvement, we should not destructure `( moduleName, name )` to then recreate such a tuple. That allocates an additional tuple unnecessarily.

(I'm sure we can find more places where we could avoid this problem)